src/entries/contentScript/views/CheckSafety.svelte

```diff
@@ -0,0 +1,64 @@
+<svelte:options tag="check-safety"/>
+
+<script lang="ts">
+    import {onMount} from "svelte";
+    import {goplus} from "~/entries/contentScript/views/network";
+
+    let currentUrl = window.location.href;
```

Comment:
if `currentUrl` is only used inside `checkSafeteCurrentUrl`, let move it into that function. Try to write pure function as possible

https://www.freecodecamp.org/news/what-is-a-pure-function-in-javascript-acb887375dfe/